### PR TITLE
Updated list of contributors and .mailmap file

### PR DESCRIPTION
**Pull request by saimn**
### Description

This pull request is to address ...

Fixes #<Issue Number>

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -204,6 +207,7 @@ Magnus Persson               <vilhelmp@gmail.com>
 Maneesh Yadav                <maneesh.yadav@sri.com>
 Mangala Gowri Krishnamoorthy <mangalagb@gmail.com>
 Marten van Kerkwijk          <mhvk@astro.utoronto.ca> <mhvk@swan.astro.utoronto.ca>
+Marten van Kerkwijk          <mhvk@astro.utoronto.ca> Marten H. van Kerkwijk <mhvk@astro.utoronto.ca>
```
Comment (pllim) [inline .mailmap:210]: This looks weird.

File: .mailmap [hunk 2]
```diff
@@ -152,6 +152,8 @@ Javier Duran                 <javier.duran@sciops.esa.int> <jduran@sciops.esa.in
 Javier Pascual Granado       <javier@iaa.es>
 Jeff Taylor                  <jeff.c.taylor@gmail.com>
 Jennifer Karr                <karr@l-145-118-237-197.leidenuniv.nl>
+Jero Bado                    <programmer@jerobado.com> <tokidokitalkyou@gmail.com>
+Jerry Ma                     <jerry.ma.nk@gmail.com> Zhiyuan  Ma <jerry.ma.nk@gmail.com>
```
Comment (pllim) [inline .mailmap:156]: Is this undoing #12694?

Comment (saimn) [reply .mailmap]: I wasn't sure which one to keep, fixed!

File: .mailmap (hunk 1 above)
Comment (saimn) [reply .mailmap:209]: Why? Without that they would appear as 2 different persons.

Comment (pllim) [reply .mailmap:209]: Oh, I thought this file only consolidate emails. Ignore me then.

**Review by pllim — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**